tree/tree/inc/ROOT/InternalTreeUtils.hxx

```diff
@@ -78,6 +78,8 @@ public:
 std::unique_ptr<TChain> MakeChainForMT(const std::string &name = "", const std::string &title = "");
 std::vector<std::unique_ptr<TChain>> MakeFriends(const ROOT::TreeUtils::RFriendInfo &finfo);
 
+std::vector<std::string> ExpandGlob(TString glob);
```

Comment:
why a TString and why by copy? :smile:

Reply:
My reasoning was:
1. Pass by value because anyway we need a copy of the basename since we are modifying it in the function (see `glob.Remove(0, slashpos + 1);`
2. TString because it might give us the chance to also provide a modern `ExpandGlob` that uses only standard C++ functionality (probably except for `TRegexp` and we could keep the same name of the function by e.g. providing an overload that takes `std::string_view`

Reply:
I see! It might be worth a comment, because this signature will make every dev that encounters it raise an eyebrow.